conference-impl/src/main/java/org/tweetwallfx/conference/impl/ConferenceClientImpl.java

```diff
@@ -62,6 +63,7 @@
 import org.tweetwallfx.conference.spi.TrackImpl;
 import org.tweetwallfx.conference.spi.util.RestCallHelper;
 import org.tweetwallfx.config.Configuration;
+import org.tweetwallfx.util.ExpiringValue;
 
 public class ConferenceClientImpl implements ConferenceClient, RatingClient {
```

Comment:
To prevent lint warning: this-escape

```suggestion
public final class ConferenceClientImpl implements ConferenceClient, RatingClient {
```